service/history/decisionHandler.go

```diff
@@ -110,7 +109,10 @@ func (handler *decisionHandlerImpl) handleDecisionTaskScheduled(
 				transferTasks:  []persistence.Task{&persistence.RecordWorkflowStartedTask{}},
 			}
 
-			startEvent, _ := msBuilder.GetStartEvent()
+			startEvent, found := msBuilder.GetStartEvent()
+			if !found {
```

Comment:
Will this check break cron workflow?

Reply:
these are missing checks. without this, code below will encounter nil pointer exception